src/pages/components/tree-view/usage.mdx

```diff
@@ -0,0 +1,356 @@
+align vertically.
+
+<Row>
+<Column colLg={12}>
+
+![Example of aligning nesting nodes](images/tree-view-usage-4.png)
+
+</Column>
+</Row>
+
+## Content
+
+### Main elements
+
+You may be able to label your nodes, or the labels may be generated from code.
+If you have control of your label names, follow these guidelines.
+
+#### Branch node label
+
+Be brief but clear while summarizing what kind of child nodes the branch node
+contains.
+
+#### Leaf node label
+
+Be brief and descriptive about what information the leaf node contains.
+
+### Overflow content
+
+If the node label is too long for one line in your tree view, add an ellipsis
+`…` and accompany with a browser-based tooltip to show the full string of text.
+
+While truncation is not preferred, you can customize the view by
+[positioning](/patterns/overflow-content/#variations) the ellipsis at the
+front-line, mid-line, or end-line of a node label depending on what information
+is most useful to the user to keep persistent in the tree view.
+
+<Row>
+<Column colLg={8}>
+
+![Example of overflowing content](images/tree-view-usage-5.png)
+
+</Column>
+</Row>
+
+### Further guidance
+
+For further content guidance, see Carbon's
+[content guidelines](/guidelines/content/overview/).
+
+## Universal behaviors
+
+### States
+
+Branch nodes and leaf nodes share the same styles for different states. The only
+difference between the structure of the two is the addition of a caret icon for
+expanding or collapsing branch nodes.
+
+<Row>
+<Column colLg={12}>
+
+![Example of node states](images/tree-view-usage-6.png)
+
+</Column>
+</Row>
+
+#### Selection indicator
+
+If the branch node of a selected child node is collapsed, the parent node should
+inherit the selected state so the user does not lose the context of what is
+selected, even if it is out of view.
+
+<Row>
+<Column colLg={12}>
+
+![Example of a selected node](images/tree-view-usage-6b.png)
+
+</Column>
+</Row>
+
+<Caption>
+  Example of selected child node when the parent is open (left) and when the
+  parent is collapsed (right).
+</Caption>
+
+### Interactions
+
+#### Focus
+
+When a single-select tree receives focus:
+
+- Focus is set on the previously selected node.
+- If none of the nodes were selected before the tree receives focus, focus is
+  set on the first node.
+
+#### Expanding and collapsing
+
+- To expand or collapse a branch node the user can click anywhere within the
+  caret icon bounding box.
+
+<Row>
+<Column colLg={8}>
+
+![Example showing click target areas for a branch node](images/tree-view-usage-7.png)
+
+</Column>
+</Row>
+
+#### Selecting
+
+- To select a branch node the user can click anywhere on the node excluding the
+  caret icon bounding box.
+- To select a leaf node the user can click anywhere on the node container.
+
+<Row>
+<Column colLg={8}>
+
+![Example showing click target areas for selecting a node](images/tree-view-usage-8.png)
+
+</Column>
+</Row>
+
+#### Keyboard controls
```

Comment:
You need a `<br />` after this header and before the table.

Reply:
I actually tried putting one in, but it kept causing ci formatting to fail ... so I just took it out lol

Reply:
ok I added it back in lets see if it will pass this time.

Reply:
nope it failed again ... dang